app/domain/authentication/authn_jwt/orchestrate_jwt_authentication.rb

```diff
@@ -6,25 +6,26 @@ module AuthnJwt
 
     OrchestrateJwtAuthentication ||= CommandClass.new(
       dependencies: {
-        token_factory: TokenFactory.new
+        jwt_configuration_factory: JwtConfigurationFactory.new,
+        jwt_authenticator: Authentication::AuthnJwt::Authenticate.new
       },
       inputs: %i[authenticator_input]
     ) do
-      extend(Forwardable)
-      def_delegators(
-        :@authenticator_input, :account, :username
-      )
 
       def call
-        authenticate
+        authenticate_jwt
       end
 
       private
 
-      def authenticate
-        @token_factory.signed_token(
-          account: account,
-          username: username
+      def vendor
+        @authenticator_input.service_id
+      end
+
+      def authenticate_jwt
+        @jwt_authenticator.call(
+          jwt_configuration: @jwt_configuration_factory.create_jwt_configuration(vendor),
+          authenticator_input: @authenticator_input
         )
       end
```

Comment:
please order the method implementations in the order they are called

Reply:
Done